src/contracts/account/ProxyAccountDeployer.sol

```diff
@@ -92,11 +92,10 @@ contract ProxyAccount is ReplayProtection {
 
 
 /**
- * A minimal relay hub contract.
- * Verifies the signer's signature and replay protection before forwarding calldata to the target.
- * Delegates nonce verification to another contract.
+ * Responsible for deploying new proxy accounts via CREATE2
+ * Every user has their own proxy account.
  */
-contract ProxyHub is ReplayProtection {
+contract ProxyAccountDeployer is ReplayProtection {
 
     // TOOD:We can remove map once we can deterministically compute
```

Comment:
TODO in code

Reply:
ya, still need to decide whether to keep mapping or not.